tomviz/PythonGeneratedDatasetReaction.cxx

```diff
@@ -370,10 +370,11 @@ void PythonGeneratedDatasetReaction::addDataset()
     layout->addWidget(buttons);
     dialog.setLayout(layout);
     dialog.layout()->setSizeConstraint(QLayout::SetFixedSize); //Make the UI non-resizeable
-
+    
```

Comment:
Ok, apparently I found a bug in Github... It told me it failed to comment on this the first few times.